composer.json

```diff
@@ -29,10 +29,9 @@
         "sonata-project/block-bundle": "^3.20",
         "sonata-project/doctrine-extensions": "^1.8",
         "sonata-project/doctrine-orm-admin-bundle": "^3.19",
-        "sonata-project/form-extensions": "^0.1.1 || ^1.4",
-        "sonata-project/notification-bundle": "^3.8",
```

Comment:
you can not remove this yet!

Reply:
Yep , I saw it. This PR will remain open, but as a draft.